client/tests/e2e/specs/test_workspaces_page.ts

```diff
@@ -16,10 +16,10 @@ describe('Check workspaces page', () => {
     cy.get('#workspace-filter-select').contains('Name');
     cy.get('#workspaces-ms-action-bar').find('#grid-view').should('have.attr', 'disabled');
     cy.get('#workspaces-ms-action-bar').find('#list-view').should('not.have.attr', 'disabled');
-    cy.get('.card').should('have.length', 2);
+    cy.get('.card').should('have.length', 3);
     cy.get('.workspace-list-item').should('have.length', 0);
     cy.get('.card').first().contains('The Copper Coronet');
-    cy.get('.card').last().contains('Trademeet');
+    cy.get('.card').last().contains("Watcher's Keep");
```

Comment:
Use single quotes

Reply:
Prettier reformats it automatically because the string contains a single quote.